pkg/controller/scyllacluster/resource_test.go

```diff
@@ -990,6 +990,49 @@ func TestStatefulSetForRack(t *testing.T) {
 									},
 								},
 							},
+							{
+								Name:            "scylladb-api-status-probe",
+								Image:           "scylladb/scylla-operator:latest",
+								ImagePullPolicy: corev1.PullIfNotPresent,
+								Command: []string{
+									"/usr/bin/scylla-operator",
+									"serve-probes",
```

Comment:
same comment, probeserver imo

Reply:
For a single command I prefer the one with a verb in it, otherwise that would have to be `scylla-operator probe-server run`,

Reply:
similar to https://github.com/scylladb/scylla-operator/blob/9518217ba718d30af4343d85959c90b43a47c32a/pkg/cmd/operator/webhooks.go#L59